charts/bundler/templates/static_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-static-data
+  namespace: {{ .Release.Namespace }}
+data:
+  {{ $currentScope := . }}
+  {{ range $path, $_ := .Files.Glob "static/*.json" }}
+  {{ . }}: |-
+    {{- with $currentScope}}
+    {{ .Files.Get $path }}
+    {{- end }}
+  {{ end }}
+
+  admin: {{ .Files.Glob "static/*.json" O}}
```

Comment:
What does `admin` do?

Reply:
I think this line was left by a previous attempt and wasn't meant to be left in